packages/conf-unwind/conf-unwind.0/opam

```diff
@@ -10,7 +10,7 @@ depexts: [
   ["libunwind-dev"] {os-family = "debian" | os-family = "ubuntu"}
   ["libunwind-devel"] {os-distribution = "centos"}
   ["libunwind-devel"] {os-distribution = "rhel"}
-  ["libunwind-devel"] {os-distribution = "fedora"}
+  ["libunwind-devel"] {os-distribution = "fedora" | os-family = "fedora"}
```

Comment:
```suggestion
  ["libunwind-devel"] {os-family = "fedora"}
```
Why not this? Can it have fedora distribution but not family?

Reply:
mmh, good question, I thought Fedora itself had `ID_LIKE = rhel` or something but it turns out they do not (maybe they had in the past?). So I guess this should be safe to do.

Reply:
I'll keep it as is, can be revisited in the future